db/create_tables.sql

```diff
@@ -21,7 +21,7 @@ CREATE TABLE IF NOT EXISTS `checkpoints` (
   `state_machine_name` VARCHAR(255) NOT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
   `state_id` BIGINT(20) NOT NULL,
-  `data` LONGTEXT,
+  `data` BLOB,
```

Comment:
We can save SM execution status and related context in this table for replay purposes.